### PR TITLE
Coto

### LINKED ISSUE #47: Coto (Argentina)
Supermarket chain in Argentina
http://www.coto.com.ar/mapassucursales/Sucursales/ListadoSucursales.aspx

### PR DESCRIPTION
**Pull request by DerekMcDaniel**
Fixes #47.

### CODE REVIEW

**Comment by iandees** [comment]
The spiders need to go in the same directory as the others, please.

**Comment by iandees** [comment]
Thanks for moving it to the correct place. Can you also lowercase the filename? Python files are usually lowercase by convention.

File: locations/spiders/Coto.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                country="Argentina",
+                phone=phone,
+                addr_full=street,
+                opening_hours="{}, {}, {}, {} ".format(
+                    mon_thu, fri, sat, sun).replace(' a ', ' - ') + (
+                    self.c_hours if self.c_hours else ''
+                )
+            )
+
+            self.c_hours = 'Community Stand: '
```
Comment (iandees) [inline locations/spiders/Coto.py]: Not sure what this is doing. Please don't store extra text between each item.

File: locations/spiders/Coto.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                ref=ref,
+                lat=lat,
+                lon=lon,
+                #  extras=extras,
+                name=name,
+                street=street,
+                country="Argentina",
+                phone=phone,
+                addr_full=street,
+                opening_hours="{}, {}, {}, {} ".format(
```
Comment (iandees) [inline locations/spiders/Coto.py]: This doesn't follow the desired opening hours format. Please see https://github.com/alltheplaces/alltheplaces/pull/254#issuecomment-349794850 for more information about the hours format.

Comment (DerekMcDaniel) [reply locations/spiders/Coto.py]: Thanks, will fix.

File: locations/spiders/Coto.py (hunk 1 above)
Comment (DerekMcDaniel) [reply locations/spiders/Coto.py]: Some stores have community stand hours. No idea what that is, but I see now it doesn't fit with desired opening hours format. These stores also have normal "branch hours" so I can remove that unless it is needed.

Comment (nicolas17) [reply locations/spiders/Coto.py]: I think that's in which times they have the office to sign up for and support the frequent-buyer rewards card. Probably not relevant for this project.

Comment (DerekMcDaniel) [reply locations/spiders/Coto.py]: Ahh! That makes sense. Thanks nicolas :)

File: locations/spiders/Coto.py [hunk 3]
```diff
@@ -1,58 +0,0 @@
-# -*- coding: utf-8 -*-
```
Comment (nicolas17) [inline locations/spiders/Coto.py]: This deleted the file, did you forget to git-add the lowercase version?

**Pull request merged**